airbytehq/airbyte
# connector-generator: remove spec reference from acceptance-test-config.yaml

## What

Closes [https://github.com/airbytehq/airbyte-issue-internal/issues/6367](https://github.com/airbytehq/airbyte-internal-issues/issues/6367)

Removes reference to a separate `spec.yaml` file from the default acceptance test configuration.

- [x] `acceptance-test-config.yaml`
- [x] Test configuration doc
- [x] `config.json` in the templates
- [x] `README.md` in the templates 

## How to test

1. Make a new python or low-code connector with `./generate.sh` script
2. Run `poetry install && poetry run source-test spec` to verify it works.
